lego/apps/users/validators.py

```diff
@@ -17,6 +17,13 @@
     "invalid",
 )
 
+github_username_validator = validators.RegexValidator(
+    r"^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$",
```

Comment:
What is this regex/where did you get it from?

Reply:
Just stole it from here: https://github.com/shinnn/github-username-regex